modules/win_mozilla_build_tester/files/pip.conf

```diff
@@ -1,4 +1,4 @@
 [global]
 disable-pip-version-check = true
 trusted-host = pypi.pub.build.mozilla.org
-download-cache = y:\pip-cache
\ No newline at end of file
+download-cache = c:\pip-cache
```

Comment:
We need to address the different cache drives that are in use. The hardware workers will be `c` but the Azure workers will be `y` and eventually `d`. Maybe convert this to a template and use a variable passed from the manifest.

Reply:
Will update in RELOPS-719